### PR TITLE
[APIS-822] In JDBC Driver, translating empty string to null when the option oracleStyleEmptyString is set.

### PR DESCRIPTION
**Pull request by byungwook-kim**
When the option oracleStyleEmptyString is set as a connection string parameter, the empty string "" should be translated to null automatically.

This modified feature is applied to the following database types as JAVA string type:
  - CHAR
  - VARCHAR
  - CLOB

### CODE REVIEW

**Comment by cubridci** [comment]
Build comment file: 
--------------
**
** This Pull Request's title is '[APIS-822] In JDBC Driver, translating empty string to null when the option oracleStyleEmptyString is set.'.
** And it does not start with a related JIRA issue number **
** Please edit this Pull Request's title. (eg: '[CBRD-20309] Redesign vacuum data flush' **
**

--------------

File: src/jdbc/cubrid/jdbc/driver/ConnectionProperties.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import java.util.Properties;
 import java.util.StringTokenizer;
 
+import cubrid.jdbc.driver.ConnectionProperties.BooleanConnectionProperty;
```
Comment (hgryoo) [inline src/jdbc/cubrid/jdbc/driver/ConnectionProperties.java:10]: unnecessary import

File: src/jdbc/cubrid/jdbc/jci/UStatement.java [hunk 2]
```diff
@@ -129,6 +131,8 @@
 	private boolean result_cacheable = false;
 	private UStmtCache stmt_cache;
 
+        private ConnectionProperties connectionProperties = new ConnectionProperties();
```
Comment (hgryoo) [inline src/jdbc/cubrid/jdbc/jci/UStatement.java:134]: it is not used anywhere

File: src/jdbc/cubrid/jdbc/driver/ConnectionProperties.java (hunk 1 above)
Comment (byungwook-kim) [reply src/jdbc/cubrid/jdbc/driver/ConnectionProperties.java:10]: Ok, I removed the line.

File: src/jdbc/cubrid/jdbc/jci/UStatement.java (hunk 2 above)
Comment (byungwook-kim) [reply src/jdbc/cubrid/jdbc/jci/UStatement.java:134]: I checked the line is unnecessary. I modified it.

**Comment by cubridci** [comment]
Build comment file: 
--------------
**
** This Pull Request's title is '[APIS-822] In JDBC Driver, translating empty string to null when the option oracleStyleEmptyString is set.'.
** And it does not start with a related JIRA issue number **
** Please edit this Pull Request's title. (eg: '[CBRD-20309] Redesign vacuum data flush' **
**

--------------

**Pull request merged**